<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PurchaseAutoInsurance</class>
 <widget class="QDialog" name="PurchaseAutoInsurance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insurance for auto</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/img/car-insurance.png</normaloff>:/img/car-insurance.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
                    background:qlineargradient(x1:0, y1:0, x2:0, y2:1, 
                          stop:0.03 rgba(184,193,201,1), 
                          stop:0.27 rgba(107,150,191,1), 
                          stop:0.92 rgba(245,39,39,1));
	font-family: &quot;Franklin Gothic Heavy&quot;;
	font-size:14pt;
}
 
QMessageBox{
	background-color: rgb(255, 255, 255);
	color:rgb(25, 25, 25);
	font: 350 12pt &quot;Bahnschrift SemiLight SemiConde&quot;;
}        
   </string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_14">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="1">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_15">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget{
	background-color:rgb(245, 245, 245);
	border:0px solid;
	border-radius:20px;
}

QComboBox{
	background-color:rgb(110, 151, 190);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	border:0px;
	border-radius:15px;
}

QComboBox:hover{
	background-color:rgb(83, 107, 135);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	border:0px solid rgb(0, 0, 0);
	border-radius:15px;
}

QLineEdit{
	background-color:rgb(110, 151, 190);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	border:0px;
	border-radius:15px;
}

QLineEdit:focus{
	background-color:rgb(83, 107, 135);
	font: 350 13pt &quot;Bahnschrift SemiLight SemiConde&quot;;
	color:rgb(1, 1, 1);
	border:2px solid rgb(0, 0, 0);
	border-radius:15px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_10">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="1">
       <spacer name="verticalSpacer_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="technicalPassportLn">
        <property name="minimumSize">
         <size>
          <width>185</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>City by technical passport:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <spacer name="verticalSpacer_9">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Franklin Gothic Heavy</family>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
color:rgba(245,39,39,1);
border:0px;
margin-top:5px;
};</string>
        </property>
        <property name="text">
         <string>AUTO INSURANCE</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer_17">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="insuranceTypeCb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="placeholderText">
         <string>Type of insurance:</string>
        </property>
        <item>
         <property name="text">
          <string>Compulsory Insurance</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All-inclusive Insurance</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="autoTypeCb">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Type of auto:</string>
        </property>
        <item>
         <property name="text">
          <string>Car</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Truck</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Motorcycle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Trailer</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="1">
       <layout class="QVBoxLayout" name="verticalLayoutSimple"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="purchasePb">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift SemiCondensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">

QPushButton{
border: solid 0px #ffffff;
background-color: rgb(83, 144, 190);;
border-radius:18px;
margin-left:63px;
margin-right:63px;
color:rgb(255, 255, 255);
}

QPushButton:hover{
border: 1.5px inset rgb(1, 1, 1);
background-color: rgb(59, 104, 136);
border-radius:18px;
}</string>
     </property>
     <property name="text">
      <string>Purchase</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <spacer name="verticalSpacer_12">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_13">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1">
    <spacer name="verticalSpacer_16">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
